spec.md

```diff
@@ -266,14 +270,15 @@ The formal definition of the SMTP STS format, using [@!RFC5234], is as follows:
                        [sts-sep sts-e]
                        [sts-sep sts-auri]
                        [sts-sep]
+                       [sts-id]
                        ; components other than sts-version and
-                       ; sts-to may appear in any order
+                       ; sts-m may appear in any order
 
     sts-version     = "v" *WSP "=" *WSP %x53 %x54 %x53 %x31
 
     sts-sep         = *WSP %x3b *WSP
 
-    sts-to          = "to" *WSP "=" *WSP ( "true" / "false" )
+    sts-m           = "to" *WSP "=" *WSP ( "enforce" / "report" )
```

Comment:
I agree, and on top of that, it looks like I jumbled up the definition of "to" in one of the Friday edits. I like "mode" better.